core/templates/person_relations.html

```diff
@@ -1,10 +1,4 @@
 <h2>Related Individuals from Your Case Management System</h2>
-<div class="container warning-container row">
```

Comment:
I didn't ask for this line to be removed. It's still important. Could you put it back?  I only wanted the line removed that you did below "For more details about service involvement click here"

Reply:
I'd assumed that the warning container was only present to contain the warning. 
Sure, I'll add it in a moment.

Reply:
Yes, but this warning is still important to have.